SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -0,0 +1,116 @@
+%global debug_package %{nil}
+Summary:        NVIDIA container runtime hook
+Name:           nvidia-container-toolkit
+Version:        1.4.2
+Release:        2%{?dist}
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/nvidia-container-runtime
+#Source0:       https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        nvidia-container-toolkit-1.4.2.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar -cf %%{name}-%%{version}-vendor.tar.gz vendor
+Source1:        %{name}-%{version}-vendor.tar.gz
+Patch0:         nvidia-container-toolkit-1.4.2.patch
+BuildRequires:  golang
+Provides:       nvidia-container-runtime-hook
```

Comment:
not entirely sure why this providers is here. it was in the original spec from nvidia. however i don't think we have anything that checks for this so it can be removed if you prefer.

Reply:
I think there's no harm in leaving the `Provides` the way it is. Since it's an external spec, there's probably a reason for it and as we bring in more of Nvidia's packages (if there are any left) we might hit avoidable build or (worse yet) run-time errors this way.

Reply:
It looks like we provide this binary. It might be okay to leave it then.